operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -306,6 +299,34 @@ public void createOrRenew(String namespace, String clusterName, Map<String, Stri
         caKeySecret = secretCertProvider.createSecret(namespace, caKeySecretName, keyData, labels, ownerRef);
     }
 
+    private boolean shouldRenew(X509Certificate currentCert) {
+        String reason = null;
+        boolean result = false;
+        if (forceRenewal) {
+            reason = "Forced";
+            result = true;
+        } else if (caCertSecret == null
+                || caCertSecret.getData().get(CA_CRT) == null) {
+            reason = "CA certificate secret " + caCertSecretName + " is missing or lacking the key " + CA_CRT;
+            result = true;
+        } else if (caKeySecret == null
+                || caKeySecret.getData().get(CA_KEY) == null) {
+            reason = "CA key secret " + caKeySecretName + " is missing or lacking the key " + CA_KEY;
+            result = true;
+        } else if (caCertSecret != null
```

Comment:
We are getting `RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE` here. The nullcheck is already done at line 308.